File: codes/go/chapter01/answer_A02.go
package main

import "fmt"

func main() {
	var N, X int
	fmt.Scan(&N, &X)

	A := make([]int, N)
	for i := 0; i < N; i++ {
		fmt.Scan(&A[i])
	}

	answer := false
	for i := 0; i < N; i++ {
		if A[i] == X {
			answer = true
			break
		}
	}

	if answer {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

File: codes/ruby/2/chapter01/answer_A02.rb
n, x = gets.split.map(&:to_i)
ar = gets.split.map(&:to_i)

answer = false
ar.each do |a|
  if a == x
    answer = true
    break
  end
end

if answer
  puts "Yes"
else
  puts "No"
end
